### PR TITLE
migrate system_config doc to another named doc

### PR DESCRIPTION
**Pull request by jamesaimonetti**

### CODE REVIEW

File: core/kazoo_config/test/kapps_config_migrate_test.erl [hunk 1]
```diff
@@ -0,0 +1,390 @@
+-module(kapps_config_migrate_test).
+
+-include_lib("eunit/include/eunit.hrl").
+
+-define(WHAPPS_CONTROLLER
+       ,<<"{
```
Comment (icehess) [inline core/kazoo_config/test/kapps_config_migrate_test.erl]: why not storing it as a fixture file and load it?

Comment (jamesaimonetti) [reply core/kazoo_config/test/kapps_config_migrate_test.erl]: Is that on 4.1?

**Comment by jamesaimonetti** [comment]
@k-anderson or @lazedo should we delete the whapps_controller (if found) once migrated? That's my only outstanding question for this PR

File: core/kazoo_config/test/kapps_config_migrate_test.erl (hunk 1 above)
Comment (icehess) [reply core/kazoo_config/test/kapps_config_migrate_test.erl]: I didn't mean FixtureDB, just save it as a JSON file in `test` directory and read/decode it in your test function.

**Pull request merged**